src/app/components/views/homepage/homepage.component.ts

```diff
@@ -67,7 +68,8 @@ export class HomepageComponent implements OnInit, OnDestroy {
       next: (loading: boolean) => {
         if(!loading && this.jobService.hasSuccessfullyCompleted()) {
           this.jobsResponse = this.jobService.getResults();
-          this.jobCount = this.jobsResponse && this.jobsResponse.jobs && Array.isArray(this.jobsResponse.jobs) ? this.jobsResponse.jobs.length : 0;
+          const jobs = this.jobsResponse?.getJobs();
+          this.jobCount = jobs && Array.isArray(jobs) ? jobs.length : 0;
```

Comment:
```suggestion
          this.jobCount = Array.isArray(jobs) ? jobs.length : 0;
```
array.isarr will care for it